optuna/integration/lightgbm_tuner/optimize.py

```diff
@@ -457,6 +448,38 @@ def best_params(self) -> Dict[str, Any]:
             params.update(self.lgbm_params)
             return params
 
+    @property
+    def best_booster(self) -> lgb.Booster:
+        """Return the best booster."""
+        if self._best_booster_with_trial_number is not None:
+            if self._best_booster_with_trial_number[1] == self.study.best_trial.number:
+                return self._best_booster_with_trial_number[0]
+        else:
+            if len(self.study.trials) == 0:
+                raise ValueError("The best booster is not available because no trials completed.")
```

Comment:
`else:` is not necessary.
```suggestion
        if len(self.study.trials) == 0:
            raise ValueError("The best booster is not available because no trials completed.")
```